distributed/tensor_parallelism/two_d_parallel_example.py

```diff
@@ -1,25 +1,23 @@
-import argparse
 
 import torch
 import torch.distributed as dist
-import torch.multiprocessing as mp
+import torch.nn as nn
+import torch.nn.functional as F
 
 from torch.distributed._tensor import DeviceMesh
 from torch.distributed.fsdp import FullyShardedDataParallel as FSDP
 from torch.distributed.tensor.parallel import (
-    PairwiseParallel,
     parallelize_module,
+    ColwiseParallel,
+    RowwiseParallel,
 )
-from torch.distributed.tensor.parallel.fsdp import enable_2d_with_fsdp
 
-from utils import cleanup, setup, ToyModel
-try:
-    from torch.distributed.tensor.parallel import (
-        SequenceParallel
-    )
-    SP_AVAILABLE = True
-except BaseException as e:
-    pass
+
+from torch.distributed._shard.sharded_tensor import ShardedTensor
+from torch.distributed._tensor import DTensor, Replicate, sharding_prop
```

Comment:
why we need to import `sharding_prop`?

Reply:
we don't - it's leftover reference.
Removed, thanks for flagging!